templates/Dockerfile-rpm.tmpl

```diff
@@ -41,9 +41,9 @@ RUN if [[ %%os%%   == oraclelinux ]] && [[ %%release%% == 7 ]]; then yum install
         && rpm -ivh lz4-devel-r131-1.el6.x86_64.rpm;  \
         else yum install -y lz4 lz4-devel; fi
 
-
 # install build tools and PostgreSQL development files
-RUN ( yum install -y https://%%rpm_url%% \
+RUN ( yum install -y https://%%rpm_url%%  \
+    && ( [[ %%pgshort%% != 15 ]] || sed -i '/\[pgdg15-updates-testing\]/{n;n;n;s/.*/enabled=1/}' /etc/yum.repos.d/pgdg-redhat-all.repo ) \
```

Comment:
good catch  New day new knowledge :)

Reply:
I tested it but it did not worked for centos 8.  Rolled back
https://github.com/citusdata/packaging/runs/7667950735?check_suite_focus=true#step:5:2894

Reply:
I am cool with the current way as long as it works.

rhel 8 uses dnf instead of yum. There may be some alternative command that we need on that distro. No need to spend more time on this I guess.